src/pages/HomeTest.tsx

```diff
@@ -0,0 +1,22 @@
+/* import { useContext, useEffect, useState } from "react";
+import { userInfo } from "../services/user.api";
+import { AuthContext } from "../context/AuthContext";
+import { IAuthContext } from "../types/types"; */
+
+const HomeTest = () => {
```

Comment:
We should not have this component in the repo. if you need it, keep it as a change in your local machine. 
Also, never submit commented code.

Reply:
done